core/quantity/README.md

```diff
@@ -2,10 +2,69 @@
 
 Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md for license terms and full copyright notice.
 
-## Description
-
 The __@itwin/core-quantity__ package contains classes for quantity formatting and parsing.
 
-## Documentation
+## Terms and Concepts
+
+### Common Terms
+
+- [Unit]($quantity)/[UnitProps]($quantity) - A named unit of measure which can be located by its name or label. The definition of any unit is represented through it's `UnitProps`.
+- [UnitsProvider]($quantity) - A class that will also locate the UnitProps for a unit given name or label. This class will also provide a [UnitConversion]($quantity) to convert from one unit to another.
```

Comment:
you say a UnitsProvider 'will also locate' and 'will also provide' ... seems wrong for the first sentence to say 'will also' because nothing comes before it.

Reply:
Fixed here and in the other QuantityFormatting.md